### PR TITLE
Group customer and for attention of into invoicee on invoices.draft and update

### PR DESCRIPTION
**Pull request by steftrenson**

### CODE REVIEW

**Review by luisaterrabrito — DISMISSED** [review]

**Review by jdrieghe — DISMISSED** [review]

**Review by lowiebenoot — APPROVED** [review]

File: src/05-invoicing/invoices.apib [hunk 1]
```diff
@@ -215,17 +220,18 @@ Update an invoice.
...
-                    + company
-            + id: `2cc40a9c-e674-43db-9b15-447ccb478200` (string)
+        + invoicee (object, required)
+            + customer (object, required)
+                + type (enum[string])
+                    + Members
+                        + contact
+                        + company
+                + id: `ebafa4c5-fa8a-4409-92e5-1b192243372f` (string)
+            + for_attention_of (object, required)
```
Comment (lowiebenoot) [inline src/05-invoicing/invoices.apib]: Shouldn't this be optional?

**Review by lowiebenoot — DISMISSED** [review]

File: src/05-invoicing/invoices.apib (hunk 1 above)
Comment (luisaterrabrito) [reply src/05-invoicing/invoices.apib]: In the update for now everything is required

Comment (WouterSioen) [reply src/05-invoicing/invoices.apib]: So if you create an invoice without a `for_attention_of`, you can't update it without adding one?

I guess this should be or optional or nullable?

Comment (jdrieghe) [reply src/05-invoicing/invoices.apib]: Right now I'm thinking `nullable` ... but then I think we should do the same in the create and make it required but nullable?

**Review by WouterSioen — DISMISSED** [review]

**Review by lowiebenoot — APPROVED** [review]

**Review by luisaterrabrito — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]

**Pull request merged**